src/lib/DB.js

```diff
@@ -30,7 +30,7 @@ class DB {
     const statementConfig = {
       resourceArn: AURORA_RESOURCE_ARN,
       secretArn: AURORA_SECRET_ARN,
-      database: AURORA_DATABASE
+      database: database ?? AURORA_DATABASE
```

Comment:
We need to allow credentials in the constructor as well. Same with `resourceArn` and `secretArn`.